### PR TITLE
Feature/limited notifictions

### PR DESCRIPTION
**Pull request by ulferts**
All existing notifications get removed if a notification of type
- success
- error
  is added.

This is for now a very naive implementation but might prove enough to satisfy the requirements as formulated in https://community.openproject.org/work_packages/20248.

In addition, it adds a success notification after wp creation.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://ulferts-feature-limited-notifictions.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://ulferts-feature-limited-notifictions.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/app/work_packages/controllers/work-package-new-controller.js [hunk 1]
```diff
@@ -120,6 +122,7 @@ module.exports = function(
 
     EditableFieldsState.save(function() {
       $rootScope.$emit('workPackagesRefreshRequired');
+      NotificationsService.addSuccess(I18n.t('js.notice_successful_create'));
```
Comment (oliverguenther) [inline frontend/app/work_packages/controllers/work-package-new-controller.js:125]: This will never be called currently since the callback in save has been removed: 
https://github.com/opf/openproject/blob/dev/frontend/app/components/inplace-edit/services/editable-fields-state.service.js#L51-L65

**Comment by oliverguenther** [comment]
I do not receive a create notification due to the comment above.

Code-wise, this looks good to me. I have a minor remark about the animation of disappearing notifications: They are moved down to make room for the new notification.

bildschirmfoto_2015-11-06_um_17_38_44_1

That's nothing critical, but may be addressed in a subsequent PR (if you agree with that looking strange).

File: frontend/app/work_packages/directives/work-package-comment-directive.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ module.exports = function(
         ctrl.writeValue
       ).then(function() {
         ctrl.discardEditing();
-        NotificationsService.addSuccess(I18n.t('js.work_packages.comment_added'));
+        NotificationsService.addSuccess(I18n.t('js.notice_successful_update'));
```
Comment (oliverguenther) [inline frontend/app/work_packages/directives/work-package-comment-directive.js:117]: Why did you change this line? If I parsed the acceptance criteria correctly, the comment success notification should be removed
